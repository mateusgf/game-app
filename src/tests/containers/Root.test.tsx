import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Root from "../../containers/Root";
import {act} from "react-dom/test-utils";


describe("Root container", () => {
  test("match snapshot", () => {

    act(() => {
      const { container } = render(<Root/>);
      expect(container).toMatchSnapshot();
    });
  });
});
