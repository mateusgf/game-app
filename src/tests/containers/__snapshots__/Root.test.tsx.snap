// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root container match snapshot 1`] = `<div />`;
